torch/_dynamo/variables/dicts.py

```diff
@@ -643,6 +667,15 @@ def call_method(
                 "for correctness we graph break when there is dict mutation and we "
                 "are trying to access a proxy object."
             )
+
+            unimplemented_v2(
+                gb_type="mapping proxy affected by dictionary mutation",
+                context=f"Source: {self.source}, Dict mutation detected",
+                explanation=msg,
+                hints=[
+                    "Avoid modifying dictionaries that might be referenced by mapping proxy objects"
```

Comment:
Add " or avoid using the mapping proxy objects after modifying its underlying dictionary".